cluster-autoscaler/main.go

```diff
@@ -323,6 +326,11 @@ func buildAutoscaler() (core.Autoscaler, error) {
 		Comparator: nodeInfoComparatorBuilder(autoscalingOptions.BalancingExtraIgnoredLabels),
 	}
 
+	opts.Processors.NodeInfoProcessor = &nodeinfos.RefineNodeInfosProcessor{
```

Comment:
I would rather just not install processor at all if the setting is disabled (instead of having the processor do no-op based on the setting). My reasons are:
 * When the setting is disabled it doesn't do anything, so why call it at all?
 * It (marginally) reduces blast radius of any bugs.
 * It's clear to the author of next NodeInfoProcessor implementation that it's safe to ignore this processor if they're not interested in the functionality. The same is not true for some other processors (ex. filterOutSchedulablePodListProcessor).